internal/graph/dotgraph.go

```diff
@@ -472,3 +472,10 @@ func min64(a, b int64) int64 {
 	}
 	return b
 }
+
+func escapeChars(s []string) []string {
```

Comment:
nit: I'd name this escapeForDot and refer to https://graphviz.org/doc/info/attrs.html#k:escString

Reply:
Done.